cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/StaticVarCompensatorTest.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.cgmes.conversion.test;
+
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.StaticVarCompensator;
+import org.junit.jupiter.api.Test;
+
+import static com.powsybl.cgmes.conversion.test.ConversionUtil.readCgmesResources;
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * @author Luma Zamarreño {@literal <zamarrenolm at aia.es>}
+ * @author José Antonio Marqués {@literal <marquesja at aia.es>}
+ */
+class StaticVarCompensatorTest {
+
+    private static final String DIR = "/staticVarCompensator/";
```

Comment:
Could you please move the test configurations files in the issues folder?
```suggestion
    private static final String DIR = "/issues/staticVarCompensator/";
```